include/model.php

```diff
@@ -786,4 +795,15 @@ protected function get_language_terms() {
 
 		return empty( $terms ) || is_wp_error( $terms ) ? array() : $terms;
 	}
+
+	/**
+	 * Tells if language objects are currently created.
+	 *
+	 * @since 3.4
+	 *
+	 * @return bool Whether language objects are created or not.
+	 */
+	public function is_creating_language_objects() {
+		return ! $this->are_language_objects_created;
+	}
```

Comment:
```suggestion
	/**
	 * Tells if language objects are currently being created.
	 *
	 * @since 3.4
	 *
	 * @return bool
	 */
	public function is_creating_language_objects() {
		return $this->is_creating_language_objects;
	}
```